RecentActivity/nbproject/project.properties

```diff
@@ -1,3 +1,4 @@
+file.reference.dd-plist-1.20.jar=C:\\Users\\kelly\\Workspace\\autopsy\\RecentActivity\\release\\modules\\ext\\dd-plist-1.20.jar
```

Comment:
Here is a hard-coded path specific to your machine.

Reply:
Sorry about that.  Fixed.